src/components/aeIdentityMain/aeIdentityMain.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div @click="forwardEvent" :class="classObject">
+    <div class="flex-row">
+      <ae-identity-avatar
+        class="ae-identity__avatar"
```

Comment:
This component already has `ae-identity-avatar` class, it is not necessary to define the similar class.

Reply:
It's better if a component doesn't need to know anything about the internals of another component. More concretely ae-identity-main does not know what classes are used in ae-identity-avatar.

Reply:
Hm, we really have this convention:
https://github.com/aeternity/aepp-components/blob/37e01ee/README.md#component-css-classes